modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -0,0 +1,107 @@
+:description: This section describes the GQLSTATUS errors that Neo4j can return, grouped by category, and an example of when they can occur.
+
+[[neo4j-gqlstatus-errors]]
+= List of all GQLSTATUS server error codes
+
+The following page provides an overview of all server errors in Neo4j, along with some scenarios and their possible solutions.
+All errors in Neo4j have severity level `ERROR`.
+
+* <<_connection-exceptions, Connection exceptions>>
+
+
+[[_connection-exceptions]]
+== Connection exceptions
+
+The following is a list of the connection exceptions that can occur in Neo4j.
+
+=== 08000
+
+Notifies the client that an exception occurred during the connection.
+
+.Error details
+[cols="<1s,<4"]
+|===
+|GQLSTATUS code
+m|0800
+|Status description
+a|error: connection exception
+|Classification
+m|CLIENT_ERROR
+|===
+
+=== 08007
+
+Notifies the client that an exception occurred during the resolution of a transaction.
+
+.Error details
+[cols="<1s,<4"]
+|===
+|GQLSTATUS code
+m|08007
+|Status description
+|error: connection exception - transaction resolution unknown.
+|Classification
+m|CLIENT_ERROR
+|===
+
+=== 08N00
+
+Notifies the client that an exception occurred while trying to connect to a specified database.
+
+.Error details
+[cols="<1s,<4"]
+|===
+|GQLSTATUS code
+m|08N00
+|Status description
+a|error: connection exception - unable to connect to database. Unable to connect to database `$db`. Unable to get bolt address of the leader. Check the status of the database. Retrying your request at a later time may succeed.
```

Comment:
For parameters, I'm not sure if it is nicer to include the formatting like **\`$db\`** or to write in the way Stefan did in the GQLSTATUS sheet **{ $db }**?  

The first way is more similar to what the user sees, while the second makes it even clearer what part is a parameter and might vary. I'm not sure which one I like better for docs, so you can decide @renetapopova and @NataliaIvakina as long as we are consistent.

Note that the formatting will vary between the parameters depending on which sort of parameter it is, for some it might be single quotes, for some back-ticks and for some it will be as is. For list it can be that if we have the error message `bla, bla, everything is wrong with { $list }`, the user will see it as`bla, bla, everything is wrong with item1` or `bla, bla, everything is wrong with item1, item2 and item3` depending on the size of the parameter list. For such a case it would be a bit unclear how to write it out in docs with option 1 maybe.

Reply:
I think the parameters should be exactly as in the code. Later, when we add a test to verify the accuracy of the codes in the docs, they should align with the code; otherwise, the test will fail.

Reply:
We can show what users will see in the examples, first example status description, and later example scenarios and returned messages.

Reply:
Ah, sorry, I see that they are { $db } in the code as well.